db/schema.rb

```diff
@@ -11,7 +11,56 @@
 #
 # It's strongly recommended to check this file into your version control system.
 
-ActiveRecord::Schema.define(:version => 20150701042203) do
+ActiveRecord::Schema.define(:version => 20160126231512) do
+
+  create_table "credence_answers", :force => true do |t|
+    t.integer  "credence_question_id"
```

Comment:
I hadn't realized you could even have a non-indexed foreign key. Or is this just treated as an FK by rails, but not implemented with a MySQL FK? In any case, I had intended it for it to be indexed, and will add one accordingly.